components/script/dom/webglrenderingcontext.rs

```diff
@@ -806,7 +806,11 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
 
         self.bound_framebuffer.set(framebuffer);
         if let Some(framebuffer) = framebuffer {
-            framebuffer.bind(target)
+            if framebuffer.is_deleted() {
```

Comment:
The spec changed and this case should be ignored now (don't change the state of the binding), plus produce an `INVALID_OP`.